webservices/resources/candidates.py

```diff
@@ -84,7 +98,9 @@ class CandidateSearch(CandidateList):
 
     schema = schemas.CandidateSearchSchema
     page_schema = schemas.CandidateSearchPageSchema
-    query_options = [sa.orm.subqueryload(models.Candidate.principal_committees)]
+    query_options = [
+        sa.orm.subqueryload(models.Candidate.principal_committees),
```

Comment:
Curious--how can you tell this is problematic with `nplusone`? It looks like tests failed with `joinedload` even with `nplusone` disabled.